src/auth/auth.service.ts

```
@@ -1,145 +1,62 @@
 import * as bcrypt from 'bcryptjs';
 import { RegisterDto } from './dto/register-user.dto';
-import { LoginDto } from './dto/login-user.dto';
-import { CreatePortfolioDto } from './dto/create-portfolio.dto';
-import { JwtService } from '@nestjs/jwt';
-import * as crypto from 'crypto';
-import { addMinutes } from 'date-fns';
-import { UserService } from 'src/user/user.service';
-import { PasswordReset } from './entities/password-reset.entity';
-import { MailService } from '../mail/mail.service';
-import { ConfigService } from '@nestjs/config';
+import { LogInDto } from './dto/loginDto';
```

ESLint

[error] 'LogInDto' is defined but never used.

(@typescript-eslint/no-unused-vars)